### DISCUSSION TITLE
Is there a way to revert to the old mode of just the current directory, instead of showing all files always?

### DISCUSSION THREAD

**Discussion by Flyingscool** [WinMerge/winmerge#2421] [General]
Hi,

It looks like there's been a recent change in that all the files are shown instead of just the files and directories of the current directory. Now with tree mode on, you click on the caret and it opens the directories, still leaving everything in the screen.  With Tree mode off, all files are shown from all directories.  When comparing directories with lots of files, this gets super cumbersome.  I prefer the old way where you could only see the files in the current directories being compared, then you'd select a directory common to both, and just see what's in it.  Can an option be added to show only the current directory?  Does it already exist and I just can't find it?

Thanks!

**Comment by sdottaka** [comment]
Try unchecking `Include subfolders` in the Select Files or Folders window, or unchecking `Include subfolders` in the Compare/Folder category of the Options window.

![image](https://github.com/user-attachments/assets/991e5873-e2b3-4ac8-925c-74c8e5d001d1)

**Comment by Flyingscool** [comment]
Yay! That worked!!! Thank you!!

Man, I was looking all over for that option.  I usually compare directories by using the context menu option, so I never see that.  Thank You!!
